src/lambda-handler.ts

```diff
@@ -0,0 +1,76 @@
+import { APIGatewayProxyEventV2 } from 'aws-lambda';
+import { Args, getS3File, resizeBuffer } from './lib.js';
+import { config } from 'process';
+import { S3ClientConfig } from '@aws-sdk/client-s3';
```

Comment:
see it, say it, sort it:

```suggestion
import { S3ClientConfig } from '@aws-sdk/client-s3';
import { APIGatewayProxyEventV2 } from 'aws-lambda';
import { config } from 'process';

import { Args, getS3File, resizeBuffer } from './lib.js';
```